src/domain/event/modal/UnenrolModal.tsx

```diff
@@ -5,32 +5,43 @@ import { useHistory } from 'react-router-dom';
 // TODO: KK-300 Check how the cancel button should look
 // TODO: KK-300 If the same, find a better/reusable location for this css module
 import { toast } from 'react-toastify';
+import { useDispatch } from 'react-redux';
 
 import unenrolOccurrenceMutation from '../mutations/unenrolOccurrenceMutation';
 import profileQuery from '../../profile/queries/ProfileQuery';
 import { unenrolOccurrenceMutationVariables } from '../../api/generatedTypes/unenrolOccurrenceMutation';
 import ConfirmModal from '../../../common/components/confirm/ConfirmModal';
+import { unenrolChild } from '../state/EventActions';
 
 interface UnenrolModalProps {
   isOpen: boolean;
   setIsOpen: (value: boolean) => void;
   childId: string;
   occurrenceId: string;
+  eventId: string;
 }
 
 const UnenrolModal: FunctionComponent<UnenrolModalProps> = ({
   isOpen,
   setIsOpen,
   childId,
   occurrenceId,
+  eventId,
 }) => {
   const history = useHistory();
   const { t } = useTranslation();
+  const dispatch = useDispatch();
 
   const [unenrolOccurrence] = useMutation<unenrolOccurrenceMutationVariables>(
     unenrolOccurrenceMutation,
     {
-      refetchQueries: [{ query: profileQuery }],
+      // FIXME: Prevent crash after unenrol:
+      // Rendered fewer hooks than expected. This may be caused by an accidental early return statement.
+      //refetchQueries: [{ query: profileQuery }],
```

Comment:
@jpsalo Do you have any idea why we get this error if we have the profileQuery refetch here?